recipes/dlib/all/conanfile.py

```diff
@@ -33,7 +34,10 @@ class DlibConan(ConanFile):
         "with_png": True,
         "with_sse2": "auto",
         "with_sse4": "auto",
-        "with_avx": "auto"
+        "with_avx": "auto",
+        "with_openblas": True,
+
+        "openblas:shared": False
```

Comment:
```suggestion
```

Better not to force unless it is needed. If there is a diamond in the dependency graph and the other side of the graph tries to force `openbla:shared=True` this will create a conflict. Unless it is needed, just leave the default and, if it is needed, better to check in `validate()` and raise if the value is not suitable.